sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestProxyTestBase.java

```diff
@@ -491,11 +507,16 @@ private void addSanitizers() {
             new TestProxySanitizer("(?:AccountKey=)(?<accountKey>.*?)(?:;)", REDACTED_VALUE, TestProxySanitizerType.BODY_REGEX).setGroupForReplace("accountKey"),
             new TestProxySanitizer("$.properties.WEBSITE_AUTH_ENCRYPTION_KEY", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY),
             new TestProxySanitizer("$.properties.DOCKER_REGISTRY_SERVER_PASSWORD", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY)
-            ));
+        ));
         sanitizers.addAll(this.sanitizers);
         interceptorManager.addSanitizers(sanitizers);
     }
 
+    private void removeSanitizers() {
+        // Remove sanitizer Location, operation-location, `id` and `name` from the list of common sanitizers.
+        interceptorManager.removeSanitizers("AZSDK2003", "AZSDK2030", "AZSDK3430", "AZSDK3493");
```

Comment:
`"AZSDK2003", "AZSDK2030", "AZSDK3430", "AZSDK3493"` -> I don't quite get what do these ids mean?

Reply:
This is by Sameeksha. I just refactored the code here.

https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs#L124

Reply:
I see, maybe we could paste the link to the comment, so that we can refer to the code of sanitizers later if we want to add/remove.

Reply:
added